nexus/db-model/src/silo_group.rs

```diff
@@ -38,3 +40,14 @@ impl SiloGroupMembership {
         Self { silo_group_id, silo_user_id }
     }
 }
+
+impl From<SiloGroup> for views::Group {
+    fn from(group: SiloGroup) -> Self {
+        Self {
+            id: group.id(),
+            // TODO the use of external_id as display_name is temporary
+            display_name: group.external_id,
+            silo_id: group.silo_id,
+        }
+    }
+}
```

Comment:
nitpick and we can address this later when we have more time, but the name `Group` is probably too generic

Reply:
do you mean maybe it should be `SiloGroup` instead of `Group`?

Reply:
Well, I never liked `SiloGroup` because it sounds like a group of silos. I was thinking more like `UserGroup`, though that's problematic because it can also contain other groups (right?)